resources/articles/sdks/android.md

```diff
@@ -13,15 +13,16 @@
 
 :body
 
-The Android SDK simplifies connecting to Schibsted account and comes in two variants. The core part of the SDK contains the login engine and APIs needed to connect to Schibsted account. You can use this to do your custom UI implementation. While the networking APIs are exposed, you use these at your own peril as no support will be given to this.
-
-The recommended way to use these SDKs is to implement the UI module. This is highly customizable and will give you complete UIs out of the box. To learn more about the SDKs, please refer to the [published documentation](https://schibsted.github.io/account-sdk-android/).
+The recommended way to use the SDK is to implement the UI module. This is highly customizable and will give you 
+complete UIs out of the box. To learn more about the SDK, please refer to the
+[published documentation](https://schibsted.github.io/account-sdk-android/).
 
 For support, please contact [schibstedaccount@schibsted.com](mailto:schibstedaccount@schibsted.com)
 
 
 ## Getting started
-To get started with either SDK, you'll need to request access to Schibsted account before you can start using them.
+To get started with the SDK, you'll need to request access to Schibsted account before you can start using it.
 This process is documented [here](https://techdocs.spid.no/selfservice/access/). 
```

Comment:
Probably it could be better to have link "in documentation" or something like this, instead of "here" for SEO reasons.
So it would look like ""
This process is documented [in techdocs](https://techdocs.spid.no/selfservice/access/).
"""